utils/misc.js

```diff
@@ -9,6 +9,57 @@ function env(name) {
   throw `env variable \`${name}\` is not set.`;
 }
 
+const AS_SECONDS = {
+  s: 1,
+  m: 60,
+  h: 60 * 60,
+  d: 60 * 60 * 24,
+  w: 60 * 60 * 24 * 7,
+};
+
+/**
+ * Convert a human readable duration string to seconds value.
+ * @param {string} duration
+ * ``` js
+ * seconds("1m") // 60
+ * seconds("1.5m") // 90
+ * ```
+ */
+function seconds(duration) {
+  if (typeof duration === "number") {
```

Comment:
nit: Marcos says, "looks a lot like a switch statement" ;)

Reply:
I'm going to remove support for number as parameter type instead :wink: